tests/integration/cartography/graph/test_cleanupbuilder.py

```diff
@@ -0,0 +1,120 @@
+from cartography.client.core.tx import load_graph_data
+from cartography.graph.job import GraphJob
+from cartography.graph.querybuilder import build_ingestion_query
+from tests.data.graph.querybuilder.sample_data.helloworld_relationships import INTERESTING_NODE_SUB_RES_ONLY
+from tests.data.graph.querybuilder.sample_data.helloworld_relationships import INTERESTING_NODE_WITH_ALL_RELS
+from tests.data.graph.querybuilder.sample_data.helloworld_relationships import MERGE_HELLO_ASSET_QUERY
+from tests.data.graph.querybuilder.sample_data.helloworld_relationships import MERGE_SUB_RESOURCE_QUERY
+from tests.data.graph.querybuilder.sample_data.helloworld_relationships import MERGE_WORLD_ASSET_QUERY
+from tests.data.graph.querybuilder.sample_models.interesting_asset import InterestingAssetSchema
+from tests.integration.util import check_nodes
+from tests.integration.util import check_rels
+
+
+def test_cleanup_interesting_asset_end_to_end(neo4j_session):
+    # Arrange: add these nodes and rels to the graph, all with lastupdated=1:
+    # (i:InterestingAsset{id:'interesting-node-id'})<-[:RELATIONSHIP_LABEL]-(:SubResource{id:sub-resource-id});
```

Comment:
TODO: Can we make a test case using a real world example instead of these fake nodes so that the test is more understandable?